code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -315,3 +314,50 @@
 	. = ..()
 	if(mounted)
 		mounted.switch_firemodes()
+
+/obj/item/clothing/gloves/tesla
+	name = "tesla gauntlet"
+	desc = "A weaponized gauntlet capable of firing lightning bolts."
+	desc_fluff = "A tesla-based weapon created by the People's Republic of Adhomai as part of their Tesla Brigade program. Because of its long recharge time, the gauntlet is commonly \
+	used as an ancillary weapon."
+	icon_state = "tesla_glove_on"
```

Comment:
can this be made a contained sprite instead?

Reply:
The gloves dmi is not a giant clusterfuck like the guns or the suit ones were, don't really think it is really needed here.